dotCMS/src/main/java/com/dotcms/contenttype/business/FieldApi.java

```diff
@@ -23,24 +52,91 @@ default FieldApi instance() {
 
 	public static final Date VALIDATE_AFTER = new Date(1451606400000l);
 	
+	/**
+	 * Retrieves the list of the base Fields Types
+	 * 
+	 * @return List of baseFieldTypes
+	 */
 	List<Class> fieldTypes();
-
+	
+	/**
+	 * Register a field type to the list of field types. Still not implemented.
+	 * 
+	 * @param type Field that wants to be registered.
+	 */
 	void registerFieldType(Field type);
 
+	/**
+	 * Removes a field type to the list of field types. Still not implemented.
+	 * 
+	 * @param type Field that wants to be removed.
+	 */
 	void deRegisterFieldType(Field type);
 
+	/**
+	 * Deletes a field in the DB. Also deletes the field variable.
```

Comment:
change to : "Deletes a field from a content type (as specified in the passed in Field object)"